app/assets/javascripts/discourse/app/components/composer-container.gjs

```diff
@@ -25,13 +26,58 @@ import htmlClass from "discourse/helpers/html-class";
 import htmlSafe from "discourse/helpers/html-safe";
 import lazyHash from "discourse/helpers/lazy-hash";
 import loadingSpinner from "discourse/helpers/loading-spinner";
+import discourseDebounce from "discourse/lib/debounce";
+import { bind } from "discourse/lib/decorators";
+import grippieDragResize from "discourse/modifiers/grippie-drag-resize";
 import { i18n } from "discourse-i18n";
 import CategoryChooser from "select-kit/components/category-chooser";
 import MiniTagChooser from "select-kit/components/mini-tag-chooser";
 
 export default class ComposerContainer extends Component {
   @service composer;
   @service site;
+  @service appEvents;
+  @service keyValueStore;
+
+  @bind
+  onResizeDragStart() {
+    this.appEvents.trigger("composer:resize-started");
+  }
+
+  @bind
+  onResizeDrag(size) {
+    this.appEvents.trigger("composer:div-resizing");
+    this.composer.set("composerHeight", `${size}px`);
+    this.keyValueStore.set({
+      key: "composerHeight",
+      value: this.composer.composerHeight,
+    });
+    document.documentElement.style.setProperty(
+      "--composer-height",
+      size ? `${size}px` : ""
+    );
+
+    this._triggerComposerResized();
+  }
+
+  @bind
+  onResizeDragEnd() {
+    this.appEvents.trigger("composer:resize-ended");
+  }
+
+  _triggerComposerResized() {
+    schedule("afterRender", () => {
```

Comment:
IMO this after render is useless, if you want it, I would do it around t`his.appEvents.trigger("composer:resized");`